.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+# Contributing a Pull Request
+
+If you haven't already, read the full [contribution guide](../CONTRIBUTING.md). Once you are done and ready to submit your PR, run through the relevant checklist below.
```

Comment:
Can you add something like this? "The contribution guide may have changed since the last time you read it."